code/style.css

```diff
@@ -1,58 +1,214 @@
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background:black;
+  color: white;
+  background: url("castle.jpg");
+  background-size: 100% 100%;
 }
 
-p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+header {
+  background: white;
+  opacity: 50%;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+  border-bottom: 1px solid;
+}
+
+header a {
+  color: black;
+  font-size: 20px;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+header h1 {
+  color: black;
+  font-size: 50px;
+}
+
+header img {
+  height: 100px;
+  margin:  0 24px 0 10px;
+}
+
+.cards p {
+  font-size: 20px;
+  margin-left: 10px;
+  font-weight: bold;
+  border-radius: 5px;
+  margin-top:0px;
+  width: 150px;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: 30px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  height: auto; 
+  background: transparent;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  flex-wrap: wrap;
+  background-image: url(assets/cluedo_cover.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: black;
+
+}
+
+.kdeck {
+  width: 280px;
+  height: 440px;
+  text-align: center;
+  padding-top: 40px;
+  background: url("frame.png");
+  background-size: 100% 100%;
+}
+
+#killer.killer-deck {
+  padding-top:5px;
 }
 
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 180px;
+  height: 350px;
+  background: white;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin-left: 50px;
+  margin-top: 20px;
 }
 
-.weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+.kdeck p{
+  font-size: 16px;
+  margin-bottom:0px;
+  }
+
+.killer-deck img {
+  width: 150px;
+  height: auto;
+}
+
+.wdeck {
+  width: 280px;
+  height: 250px;
   text-align: center;
   padding-top: 30px;
+  margin: 10px;
+  background: url("frame-2.png");
+  background-size: 100% 100%;
+}
+
+.wdeck p{
+  font-size: 13px;
+  margin-bottom:0px;
+}
+
+.weapon-deck img {
```

Comment:
Some of the weapon pictures takes a bit longer to load than the rest.
Are some of them larger?